docs/src/threading.rst

```diff
@@ -131,8 +131,13 @@ Functions return 0 on success or an error code < 0 (unless the
 return type is void, of course).
 
 .. note::
-    Callers should be prepared to deal with spurious wakeups on :c:func:`uv_cond_wait` and
-    :c:func:`uv_cond_timedwait`.
+    1. Callers should be prepared to deal with spurious wakeups on :c:func:`uv_cond_wait`
+       and :c:func:`uv_cond_timedwait`.
+    2. On z/OS, the time to wait for :c:func:`uv_cond_timedwait` is specified by the timeout
+       parameter as an absolute system time at which the wait expires. If the current system
```

Comment:
The first line is at odds with the implementation (or the wording is confusing) because the timeout argument is relative to the current time, it's not an absolute timestamp.

Reply:
fixed